packages/frontend/src/redux/actions/account.js

```diff
@@ -107,29 +137,29 @@ export const handleRefreshUrl = (prevRouter) => (dispatch, getState) => {
             dispatch(refreshUrl(loadState()));
         }
         dispatch(handleFlowLimitation());
-        const { transactions, callbackUrl, meta } = getState().account.url;
+        const { transactions, callbackUrl, meta } = selectAccountUrl(getState());
         if (transactions) {
             dispatch(parseTransactionsToSign({ transactions, callbackUrl, meta }));
         }
     }
 };
 
 const checkContractId = () => async (dispatch, getState) => {
-    const { contract_id, failure_url } = getState().account.url;
+    const { contractId, failureUrl } = selectAccountUrl(getState());
```

Comment:
It looks like these names would need to stay snake_cased (or be renamed in the destructure), but maybe it's even better to double down on the selectors and add a `selectAccountUrlFailureUrl` selector and get the values for these two separately, something like
```
const contractId = selectAccountUrlContractId(getState());
const failureUrl = selectAccountUrlFailureUrl(getState());
```

We now have selectors for two other URLs on the account URL object, would it make sense to add one for `failure_url` as well?

Reply:
Wow, what a stupid bug from my side... Thanks for catching!